crates/solidity/outputs/cargo/tests/src/lib.rs

```diff
@@ -10,5 +10,8 @@ mod compilation;
 mod cst_output;
```

Comment:
nit: I wonder if we should group submodules here into same same parents as the product?

- backend:
- bindings:
  - binding_resolver
  - binding_rules
  - bindings_output
- compilation:
  - resolver
- cst:
  - cst_output
  - multi_part_file
  - parser
  - queries
  - trivia
  - utils

Reply:
This is a major refactor of tests. I would normally suggest to tackle that in a different PR, but I'm going to instead add it to this PR's description (the idea is good).